pkg/controllers/machine/suite_test.go

```diff
@@ -761,11 +764,37 @@ var _ = Describe("Controller", func() {
 			machine = ExpectExists(ctx, env.Client, machine)
 
 			// If the node hasn't registered in the liveness timeframe, then we deprovision the Machine
-			fakeClock.Step(time.Minute * 11)
+			fakeClock.Step(time.Minute * 20)
 			ExpectReconcileSucceeded(ctx, machineController, client.ObjectKeyFromObject(machine))
 			ExpectReconcileSucceeded(ctx, machineController, client.ObjectKeyFromObject(machine)) // Reconcile again to handle termination flow
 			ExpectNotFound(ctx, env.Client, machine)
 		})
+		It("should not delete the Machine when the Node hasn't registered to the Machine past the liveness TTL if ttlAfterNotRegistered is disabled", func() {
+			s := test.Settings()
+			s.TTLAfterNotRegistered = nil
+			ctx = settings.ToContext(ctx, s)
+			machine := test.Machine(v1alpha5.Machine{
```

Comment:
musing: would be nice to make this `machineOptions` so you can de-dupe some of the common initialization code.

Reply:
Not sure what you mean here. I think the better solution is to just initialize a default machine in the BeforeEach. Realistically, though, there's enough difference between these machines in testing that I don't think that it's called for here.